wscript

```diff
@@ -71,6 +71,16 @@ compiler_flags_dictionaries= {
         'generic-arm64' : '',
         # All flags required to get basic warnings to be generated by the compiler
         'basic-warnings' : [ '-Wall', '-Wpointer-arith', '-Wcast-qual', '-Wcast-align', '-Wno-unused-parameter' ],
+        # All flags for a quiet build
+        'quiet-build' : [ '-Wno-deprecated-declarations', '-Wno-unused-value', '-Wno-unused-result',
```

Comment:
Wow, I really think a quiet build should just set`Wno-deprecated-declarations` and `Wno-unused-but-set-variable'`, and the rest should be fixed.  Perhaps `Wno-unknown-pragmas` can remain as well.

Reply:
how about quiet and really-quiet?

On macOS there are many deprecated warnings (since openGL is deprecated, but it still works), and in some cases we do have "unused values" e.g.  table  `++row` counters. which are fine. So we can safely hide those.

Reply:
I just had lots of gtk warnings about their own deprecations and was trying to get an overview about what else is warned about. The nice thing about the quiet-build approach is that you can remove one warning at a time and fix it because it's only a few cases per warning (instead of hundreds of errors flying by). The nice thing about Werror would be that no fixed warning would ever be introduced again by new code.